test/language/statements/async-function/evaluation-this-value-global.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+es7id: pending
+description: >
+  The this value is set to the global object when not passed in sloppy mode.
+flags: [noStrict]
+---*/
+
+var glob = Function("return this;")();
```

Comment:
Could you use the `fnGlobalObject.js` helper here? Using that consistently allows implementers to replace with a host-specific equivalent in cases where the Function constructor is disallowed.